A3-channel-tracing.md

```diff
@@ -50,10 +50,15 @@ We define a minimal set of events that must be traced, in order to consider the
 All of these events must be traced:
 - Channel creation and deletion
 - Subchannel creation and deletion
-- Address resolution resulting from service config change
-- Address resolution that causes number of backends to go from zero to non-zero
 - Channel connectivity state changes
 - Subchannel connectivity state changes
+- Interesting address resolution events (see below for details)
+
+Address resolution is special case. We want to track resolution events, but we do not want to flood the trace buffer with them. So we define several types of "interesting" resolution events that must be traced:
+- Address resolution resulting in service config change
+- Address resolution that causes number of backends to go from zero to non-zero
+- Address resolution that causes number of backends to go from non-zero to zero
+- Address resolution that causes a new LB policy to be created
```

Comment:
Is it possible that "Address resolution that causes a new LB policy to be created" without "resulting in service config change"? What is the case?

Reply:
Yes.  If the resolver had previously not returned any balancer addresses and now returns one, we will switch to the grpclb LB policy.  Conversely, if the resolver had previously returned a balancer address and now does not return any, then we will switch away from the grpclb LB policy.

Reply:
I see.